src/app/component/MissionCard.jsx

```diff
@@ -76,13 +90,72 @@ const MissionCard = withStyles(styles)(({ children, classes, mission, ...rest })
 
   return (
     <Card className={classes.root} {...rest}>
-      <CardHeader
-        title={title}
-        titleTypographyProps={{ variant: "h2", component: "span" }}
-        className={classes.cardHeader}
-      />
       <CardContent className={classes.cardContent}>
-        <MissionCardContent contentItems={contentItems} classes={classes} />
+        <Grid container spacing="1" alignItems="flex-start" justify="flex-end" direction="row">
+          <Grid item>
+            <img height="20" src={appleIcon} alt="" />
+          </Grid>
+          <Grid item style={{ flex: 1 }}>
+            {title}
+          </Grid>
+          <Grid item>
+            <InfoIcon />
+          </Grid>
+        </Grid>
+      </CardContent>
+      <CardContent className={classes.cardContent}>
+        <Grid container direction="row">
+          <Grid item className={classes.halfRow}>
+            <Grid container direction="column">
+              <Grid item>
+                <Grid container direction="row">
+                  <Grid item>
+                    <PublishIcon />
+                  </Grid>
+                  <Grid item>
+                    <Typography variant="h5" noWrap>
+                      PICK UP
+                    </Typography>
+                  </Grid>
+                </Grid>
+              </Grid>
+              <Grid item>{mission.pickUpLocation.label}</Grid>
+              <Grid item>
+                <a href={`https://www.google.com/maps/dir/"${location}"`}>{location}</a>
```

Comment:
Is it possible to 'truncate' the location text so that it ends in "..." if its too long? (per the design)

Reply:
@laredotornado  [Material UI Typography](https://material-ui.com/api/typography/): `noWrap` true

Reply:
@utunga @karenkun, Changes are in to create the ellipsis when text would normally wrap.